ocd_backend/transformers/ibabs_report.py

```diff
@@ -117,7 +117,7 @@ def report_item(self, content_type, raw_item, canonical_iri, cached_path, **kwar
         attachment_file.size_in_bytes = document['FileSize']
         attachment_file.name = document['DisplayName']
         attachment_file.is_referenced_by = report
-        if report.start_date:
+        if report.values.get('start_date'):
```

Comment:
I think it is better to do it like `if 'start_date' in report and report.start_date`.
Otherwise obscurities might arise that I can't give an example of but of which we have seen quite a lot in this project already.